modules/installation/pages/upgrade.adoc

```diff
@@ -50,4 +50,9 @@ to check the feasibility of support the range of earlier versions.
 Upgrade or migration from the Developer Edition is not supported.
 
 For detailed upgrade procedures, see
-https://tigergraph.freshdesk.com/support/solutions/articles/5000859240-2-x-to-3-0-0-upgrade-flow
\ No newline at end of file
+https://tigergraph.freshdesk.com/support/solutions/articles/5000859240-2-x-to-3-0-0-upgrade-flow
+
+[[upgrading-cloud-marketplace-image]]
```

Comment:
Also need to add instructions for upgrading from previous 3.x edition of marketplace images. All that needs to be done is use GBar to backup the old instance and restore it on the new one.
1. Backup old marketplace instance using gbar backup.
2. Start an instance with the latest Cloud Marketplace listing
3. Restore the old backup on to the new listing
You may also want to add a reference to the Gbar documentation for more information.